### PR TITLE
Fixed typos and style in string interpolation sections

### PR DESCRIPTION
**Pull request by pkulikov**
Fixed typos ("compile-time" as adjective goes with dash) and style (use present tense instead of future).
Also added links connecting various pages on string interpolation in the documentation.

### CODE REVIEW

File: docs/csharp/tutorials/string-interpolation.md [hunk 1]
```diff
@@ -55,7 +55,7 @@ To execute the program, use `dotnet run`. You should see "Hello, World" output t
 
 ## Intro to String Interpolation
 
-With `System.String.Format`, you specify "placeholders" in a string that are replaced by the parameters following the string. For instance:
+With <xref:System.String.Format%2A?displayProperty=nameWithType>, you specify "placeholders" in a string that are replaced by the parameters following the string. For instance:
```
Comment (rpetrusha) [inline docs/csharp/tutorials/string-interpolation.md]: nit: these are arguments, not parameters.

File: docs/csharp/tutorials/string-interpolation.md [hunk 2]
```diff
@@ -110,21 +110,19 @@ var adj = "quick";
...
 
 ## Localization and Internationalization
 
-An interpolated string supports `IFormattable` and `FormattableString`, which can be useful for internationalization.
+An interpolated string supports <xref:System.IFormattable?displayProperty=nameWithType> and <xref:System.FormattableString?displayProperty=nameWithType>, which can be useful for internationalization.
 
-By default, an interpolated string uses the current culture. To use a different culture, you could cast it as `IFormattable`
-
-For instance:
+By default, an interpolated string uses the current culture. To use a different culture, you could cast it as `IFormattable`. For instance:
```
Comment (rpetrusha) [inline docs/csharp/tutorials/string-interpolation.md]: nit: delete "could"

File: docs/csharp/tutorials/string-interpolation.md [hunk 3]
```diff
@@ -110,21 +110,19 @@ var adj = "quick";
...
 
 [!code-csharp[Interpolation internationalization example](../../../samples/snippets/csharp/new-in-6/string-interpolation.cs#InterpolationInternationalizationExample)]  
 
 ## Conclusion 
 
-In this tutorial, you learned how to use string interpolation features of C# 6. It's basically a more concise way of writing simple `String.Format` statements, with some caveats for more advanced uses of it.
+In this tutorial, you learned how to use string interpolation features of C# 6. It's basically a more concise way of writing simple `String.Format` statements, with some caveats for more advanced uses of it. For more information, see the [Interpolated Strings](../../csharp//language-reference/keywords/interpolated-strings.md) topic.
```
Comment (rpetrusha) [inline docs/csharp/tutorials/string-interpolation.md]: nit: delete "of it"

**Review by rpetrusha — CHANGES_REQUESTED** [review]
Thanks for making these improvements, @pkulikov. Once again, there are a number of nits that are unrelated to your PR and that I'm hoping you wouldn't mind addressing.

**Comment by pkulikov** [comment]
@rpetrusha I've addressed your feedback, thank you.

**Review by rpetrusha — APPROVED** [review]

**Comment by rpetrusha** [comment]
Thanks for making the additional changes, @pkulikov. We'll merge your PR now. The changes should go live today or tomorrow.

**Pull request merged**